core/controllers/feedback.py

```diff
@@ -33,13 +33,35 @@ class ThreadListHandler(base.BaseHandler):
 
     @acl_decorators.can_play_exploration
     def get(self, exploration_id):
+        feedback_thread_dicts = [
+            thread.to_dict() for thread in feedback_services.get_all_threads(
+                feconf.ENTITY_TYPE_EXPLORATION, exploration_id, False)
+        ]
+
+        for thread_dict in feedback_thread_dicts:
+            thread_dict['messages'] = [
+                message.to_dict() for message in feedback_services.get_messages(
```

Comment:
This is going to be a lot of GET operations. Is it possible to do a single get-multi instead (for multiple threads)?

Reply:
Don't think the model supports it yet but I'll look into it!